### PR TITLE
Ft fix bugs from develop

### LINKED ISSUE #32: On develop branch there is an error "MODULE_NOT_FOUND"
Seems like the latest version on develop branch has been interrupted.
You can open a new branch to fix this issue.

### PR DESCRIPTION
**Pull request by Muhet**
#### Description of Task to be completed?
          To fix the conflict which was found in main branch
#### How should this be manually tested?
           everyone must check his/her task if it is works
#### Questions:

### CODE REVIEW

**Review by NTElissa — COMMENTED** [review]
squash commits

File: src/routes/authRoutes.js [hunk 1]
```diff
@@ -37,26 +40,26 @@ router.get('/callback', (req, res) => {
...
+  passport.authenticate('google', {
     session: false,
-    failureRedirect: "/",
+    failureRedirect: '/',
   }),
-  googleAuthHandler
+  googleAuthHandler,
 );
 
 router.post('/logout', logout);
```
Comment (nili-alu) [inline src/routes/authRoutes.js:55]: vendor route missed 
add these lines:
`import vendor from '../controller/vendorController.js';

router.post('/vendor',isAdmin, vendor);`

**Review by nili-alu — APPROVED** [review]

File: src/routes/authRoutes.js (hunk 1 above)
Comment (nili-alu) [inline src/routes/authRoutes.js:60]: vendor route missed, add the two lines:
import vendor from '../controller/vendorController.js';

router.post('/vendor',isAdmin, vendor);

**Review by nili-alu — CHANGES_REQUESTED** [review]

**Review by ALVINdimpos — APPROVED** [review]

**Review by vanessa-abewe — APPROVED** [review]

**Review by faid-terence — APPROVED** [review]

**Review by NTElissa — APPROVED** [review]
good progress

**Review by bryan936M — APPROVED** [review]
All my codes are there

**Review by kagaba-etienne — COMMENTED** [review]
Nice

**Review by kagaba-etienne — APPROVED** [review]
Nice

**Review by yobuir — APPROVED** [review]

**Review by nili-alu — APPROVED** [review]

**Review by chanceDira — APPROVED** [review]
Good work

**Pull request merged**